### PR TITLE
v1.0.2

### PR DESCRIPTION
**Pull request by chrismatix**
- Babel (gulpfile.js -> gulpfile.babel.js)
- path config file

>Changed
- build process now fully managed by gulp

>Removed
- build filey

### CODE REVIEW

File: gulpfile.babel.js [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        .pipe(gulp.dest('./public'))
+});
+
+// Watch asset folder for changes
+gulp.task('watch', ['scss', 'images', 'js'], function () {
+    gulp.watch(srcPaths.SCSS, ['scss']);
+    gulp.watch(srcPaths.IMAGES, ['images']);
+    gulp.watch(srcPaths.JS, ['js']);
+
+    const hugoServer = spawn('hugo', ['server', '-v']);
```
Comment (skew202) [inline gulpfile.babel.js:100]: Nice!

File: gulpfile.babel.js [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+
+    hugoServer.stderr.on('data', (data) => process.stderr.write(data));
+
+    hugoServer.on('close', (code) => {
+        console.log(`hugo exited with code ${code}`);
+    });
+});
+
+gulp.task('build', function(callback) {
+    runSequence(
```
Comment (skew202) [inline gulpfile.babel.js:112]: one clearly can use gulp in theses `wepack3 days`

File: package.json [hunk 3]
```diff
@@ -1,24 +1,29 @@
...
     "cssnano": "^3.10.0",
     "del": "^3.0.0",
     "gulp": "^3.9.1",
     "gulp-babel": "^6.1.2",
     "gulp-hash": "^4.1.1",
+    "gulp-htmlmin": "^3.0.0",
     "gulp-postcss": "^7.0.0",
     "gulp-sass": "^3.1.0",
-    "node-sass": "^4.5.3"
+    "node-sass": "^4.5.3",
```
Comment (skew202) [inline package.json]: I was wrong about adding node-sass as an explicit dep. `gulp-sass` brings it in anyway.
Maybw it was due to npm > 5 not installing those any longer...

What about adding 
``` json
  "engines": {
    "npm": "~5.0.0",
    "node": ">=7.0.0<7.10.0"
  },
```

**Review by skew202 — APPROVED** [review]

**Comment by chrismatix** [comment]
closes #4

**Pull request merged**